hack/release/prepare-assets.sh

```diff
@@ -37,21 +37,23 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 pushd $THIS_DIR/../.. > /dev/null
 
-ANTCTL_BUILDS=(
+ANTREA_BUILDS=(
     "linux amd64 linux-x86_64"
     "linux arm64 linux-arm64"
     "linux arm linux-arm"
     "windows amd64 windows-x86_64.exe"
     "darwin amd64 darwin-x86_64"
 )
 
-for build in "${ANTCTL_BUILDS[@]}"; do
+for build in "${ANTREA_BUILDS[@]}"; do
     args=($build)
     os="${args[0]}"
     arch="${args[1]}"
     suffix="${args[2]}"
 
     GOOS=$os GOARCH=$arch ANTCTL_BINARY_NAME="antctl-$suffix" BINDIR=$1/ make antctl-release
+    cd ./plugins/octant && GOOS=$os GOARCH=$arch ANTREA_OCTANT_PLUGIN_BINARY_NAME="antrea-octant-plugin-$suffix" \
+    BINDIR=../../$1/ make antrea-octant-plugin-release && cd ../..
```

Comment:
That doesn't seem right. What if `$1` is an absolute path?
Maybe a good solution would be to ensure that we have an absolute path and just use that. With something like this at the top, which should be cross-platform:
```
OUTPUT_DIR=$(cd "$1"; pwd)
...
GOOS=$os GOARCH=$arch ANTCTL_BINARY_NAME="antctl-$suffix" BINDIR="$OUTPUT_DIR" make antctl-release
```

Reply:
Thanks for the comments. Have updated accordingly, please help review the latest change.